src/hydrate/runner/render.ts

```diff
@@ -48,7 +48,7 @@ export function renderToString(
   /**
    * Defines whether we render the shadow root as a declarative shadow root or as scoped shadow root.
    */
-  opts.serializeShadowRoot = typeof opts.serializeShadowRoot === 'boolean' ? opts.serializeShadowRoot : true;
+  opts.serializeShadowRoot = typeof opts.serializeShadowRoot === 'undefined' ? 'dsd' : opts.serializeShadowRoot;
```

Comment:
Shall we put the `serializeShadowRoot` options into an enum? Makes it easier to reuse them.

Reply:
hm I'm not sure how / where to put that as that will change our public type definitions files as having some actual exported JS ... might be a bit risky?

Reply:
good point, let's keep it simple for now.